Engine/ScriptAnalyzerEngine.csproj

```diff
@@ -44,6 +48,9 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>..\..\..\..\..\..\fbl_srv2_ci_mgmt.binaries.amd64chk\monad\System.Management.Automation.dll</HintPath>
```

Comment:
I think we must remove the hintpath setting. SMA.dll needs to be always picked up from GAC.